File: fixtures/scrapy_output/get_with_all_numerical_data.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        headers = {
            'Content-Type': 'application/json',
            'Accept': 'application/json',
        }
        body = '18233982904'
        yield Request(
            'http://198.30.191.00:8309/CurlToNode',
            callback=self.parse,
            method='POST',
            headers=headers,
            body=body,
        )

    def parse(self, response):
        pass

File: test/fixtures/node-axios/get_with_all_numerical_data.js
import axios from 'axios';

const response = await axios.post(
  'http://localhost:28139/CurlToNode',
  '18233982904',
  {
    headers: {
      'Content-Type': 'application/json',
      'Accept': 'application/json'
    }
  }
);
